tests/src/python/test_qgsdatumtransforms.py

```diff
@@ -0,0 +1,148 @@
+        self.assertTrue(ops[1].grids[0].directDownload)
+        self.assertTrue(ops[1].grids[0].openLicense)
+        self.assertEqual(ops[2].name, 'GDA94 to GDA2020 (3)')
+        self.assertEqual(ops[2].proj, '+proj=pipeline +step +proj=axisswap +order=2,1 +step +proj=unitconvert +xy_in=deg +xy_out=rad +step +proj=hgridshift +grids=GDA94_GDA2020_conformal.gsb +step +proj=unitconvert +xy_in=rad +xy_out=deg +step +proj=axisswap +order=2,1')
+        self.assertEqual(ops[2].accuracy, 0.05) # actually incorrect in EPSG registry, may need updating
+        self.assertEqual(len(ops[2].grids), 1)
+        self.assertEqual(ops[2].grids[0].shortName, 'GDA94_GDA2020_conformal.gsb')
+        self.assertEqual(ops[2].grids[0].fullName, '')
+        self.assertEqual(ops[2].grids[0].packageName, 'proj-datumgrid-oceania')
+        self.assertEqual(ops[2].grids[0].url, 'https://download.osgeo.org/proj/proj-datumgrid-oceania-latest.zip')
+        self.assertTrue(ops[2].grids[0].directDownload)
+        self.assertTrue(ops[2].grids[0].openLicense)
+        self.assertEqual(ops[3].name, 'GDA94 to GDA2020 (5)')
+        self.assertEqual(ops[3].proj, '+proj=pipeline +step +proj=axisswap +order=2,1 +step +proj=unitconvert +xy_in=deg +xy_out=rad +step +proj=hgridshift +grids=GDA94_GDA2020_conformal_cocos_island.gsb +step +proj=unitconvert +xy_in=rad +xy_out=deg +step +proj=axisswap +order=2,1')
+        self.assertEqual(ops[3].accuracy, 0.05) # actually incorrect in EPSG registry, may need updating
+        self.assertEqual(len(ops[3].grids), 1)
+        self.assertEqual(ops[3].grids[0].shortName, 'GDA94_GDA2020_conformal_cocos_island.gsb')
+        self.assertEqual(ops[3].grids[0].fullName, '')
+        self.assertEqual(ops[3].grids[0].packageName, '')
+        self.assertEqual(ops[3].grids[0].url, '')
+        self.assertFalse(ops[3].grids[0].directDownload)
+        self.assertFalse(ops[3].grids[0].openLicense)
+        self.assertEqual(ops[4].name, 'GDA94 to GDA2020 (4)')
+        self.assertEqual(ops[4].proj, '+proj=pipeline +step +proj=axisswap +order=2,1 +step +proj=unitconvert +xy_in=deg +xy_out=rad +step +proj=hgridshift +grids=GDA94_GDA2020_conformal_christmas_island.gsb +step +proj=unitconvert +xy_in=rad +xy_out=deg +step +proj=axisswap +order=2,1')
+        self.assertEqual(ops[4].accuracy, 0.05) # actually incorrect in EPSG registry, may need updating
+        self.assertEqual(len(ops[4].grids), 1)
+        self.assertEqual(ops[4].grids[0].shortName, 'GDA94_GDA2020_conformal_christmas_island.gsb')
+        self.assertEqual(ops[4].grids[0].fullName, '')
+        self.assertEqual(ops[4].grids[0].packageName, '')
+        self.assertEqual(ops[4].grids[0].url, '')
+        self.assertFalse(ops[4].grids[0].directDownload)
+        self.assertFalse(ops[4].grids[0].openLicense)
+
+        # uses a pivot datum
+        ops = QgsDatumTransform.operations(QgsCoordinateReferenceSystem('EPSG:3111'),
```

Comment:
this part ('uses a pivot datum') doesn't really add more testing of the QGIS side of things than the above one. This is more a PROJ test